caikit/runtime/__init__.py

```diff
@@ -13,4 +13,8 @@
 # limitations under the License.
 
 # Local
-from .service_factory import get_inference_request, get_train_params, get_train_request
+from .grpc_server.service_factory import (
```

Comment:
Hm, I'm still confused here. The `service_factory` stuff seems like it should _not_ be specific to the `grpc_server` since it is also required by the `http_server` in order to wrap the deduced services into HTTP APIs?

Reply:
Yeaaah, that's unfortunately because of the fact that in `test_service_factory`, we use the `runtime_grpc_server` fixture, which can only be available within `grpc_server` package. And since we want to mirror the code to the tests, I'm stuck with this